Makefile.am

```diff
@@ -31,14 +31,16 @@ DIST_SUBDIRS += jimtcl
 endif
 
 # common flags used in openocd build
-AM_CFLAGS = $(GCC_WARNINGS)
+AM_CFLAGS = $(GCC_WARNINGS)\
+			  -DFD_SETSIZE=128
 
 AM_CPPFLAGS = $(HOST_CPPFLAGS)\
 			  -I$(top_srcdir)/src \
 			  -I$(top_builddir)/src \
 			  -I$(top_srcdir)/src/helper \
 			  -DPKGDATADIR=\"$(pkgdatadir)\" \
-			  -DBINDIR=\"$(bindir)\"
+			  -DBINDIR=\"$(bindir)\"\
+			  -DFD_SETSIZE=128
```

Comment:
I am curious what was the reason behind this change.

In what scenario was the original socket limit reached? (I assume that the [default limit for connections](https://docs.microsoft.com/en-us/windows/win32/winsock/maximum-number-of-sockets-supported-2) prior this change was 64.)

When OpenOCD is used in a typical way, I would expect one socket for the GDB connection, possibly one socket for the connection towards the target (e.g. remote_bitbang), plus one connection for the telnet/tcl console. What consumed the 64 connection slots?

Reply:
I'm curious too.

In normal use I'd expect one each for gdb server/remote serial protocol interface, Tcl interface and telnet interface. Maybe some "JTAG" interfaces might use another? But 3-4 seems like it should be the max for a single instance of openocd? Unless the change is to allow more than one instance of openings to use a wider range of ports of something?